Gifski/Preview/PreviewRendererContext.swift

```diff
@@ -92,36 +101,48 @@ struct PreviewRendererContext {
 		samplerDescriptor.tAddressMode = .clampToEdge
 
 		guard let samplerState = metalDevice.makeSamplerState(descriptor: samplerDescriptor) else {
-			throw PreviewRenderer.Error.failedToMakeSampler
+			throw SetupSamplerStateError.failedToMakeSampler
```

Comment:
In cases like this, it was better to use `PreviewRenderer.Error.failedToMakeSampler`